index.js

```diff
@@ -189,7 +189,9 @@ function addComponent(
       return;
     }
     let version = pkg.version;
-    let licenses = utils.getLicenses(pkg, format);
+    //TODO: cherry-pick from upstream optional license fetching
+    //let licenses = utils.getLicenses(pkg, format);
```

Comment:
No this does not disable license fetching. This only stops the information from getting added to BoM after the fetch is already complete

Reply:
https://github.com/ShiftLeftSecurity/cdxgen/blob/master/utils.js#L905 - change ```if (!process.env.SKIP_FETCH_LICENSE) {``` to ```if (process.env.FETCH_LICENSE) {``` so it becomes opt-in

Reply:
And added a similar check here as well to speed up csharp https://github.com/ShiftLeftSecurity/cdxgen/blob/master/utils.js#L1184

Reply:
:+1:

Reply:
So this one can be closed?

Reply:
yes